### ISSUE TITLE
Investigate `data reader can read partition values` test failure in CI

### ISSUE THREAD

**Issue by zsxwing** [delta-io/connectors#258]
After upgrading SBT to 1.6.1, `data reader can read partition values` starts to fail. But I cannot reproduce it locally. 

```
[info] - data reader can read partition values *** FAILED ***
[info]   java.io.FileNotFoundException: File file:/home/runner/work/connectors/connectors/standalone/target/scala-2.12/test-classes/golden/data-reader-partition-values/as_int=0/as_long=0/as_byte=0/as_short=0/as_boolean=true/as_float=0.0/as_double=0.0/as_string=0/as_string_lit_null=null/as_date=2021-09-08/as_timestamp=2021-09-08 11%3A11%3A11/as_big_decimal=0/part-00000-b9dc86ae-0134-4363-bd87-19cfb3403e9a.c000.snappy.parquet does not exist
[info]   at org.apache.hadoop.fs.RawLocalFileSystem.deprecatedGetFileStatus(RawLocalFileSystem.java:641)
[info]   at org.apache.hadoop.fs.RawLocalFileSystem.getFileLinkStatusInternal(RawLocalFileSystem.java:930)
[info]   at org.apache.hadoop.fs.RawLocalFileSystem.getFileStatus(RawLocalFileSystem.java:631)
[info]   at org.apache.hadoop.fs.FilterFileSystem.getFileStatus(FilterFileSystem.java:454)
[info]   at org.apache.parquet.hadoop.ParquetReader$Builder.build(ParquetReader.java:300)
[info]   at com.github.mjakubowski84.parquet4s.ParquetIterableImpl$$anon$2.<init>(ParquetReader.scala:103)
[info]   at com.github.mjakubowski84.parquet4s.ParquetIterableImpl.iterator(ParquetReader.scala:100)
[info]   at io.delta.standalone.internal.data.CloseableParquetDataIterator.<init>(CloseableParquetDataIterator.scala:69)
[info]   at io.delta.standalone.internal.SnapshotImpl.open(SnapshotImpl.scala:87)
[info]   at io.delta.standalone.internal.DeltaDataReaderSuite.$anonfun$new$36(DeltaDataReaderSuite.scala:311)
[info]   ...
```

We disabled the test to merge SBT upgrade PR. This ticket is for investigate the issue and re-enable the test.

**Comment by allisonport-db** [comment]
It works if you run `standalone/compile` then `copyResources` then `standalone/testOnly *DeltaDataReaderSuite`, still not sure what the problem is however

**Comment by allisonport-db** [comment]
This is also new?
```
Warning: Tried to extract the base path for relative glob golden-tables/src/test/resources/**/*. To disable this warning, re-run the program with java option, -Dsbt.io.implicit.relative.glob.conversion=allow
Warning: Tried to extract the base path for relative glob golden-tables/src/test/resources/**/*. To disable this warning, re-run the program with java option, -Dsbt.io.implicit.relative.glob.conversion=allow
Warning: Tried to extract the base path for relative glob golden-tables/src/test/resources/**/*. To disable this warning, re-run the program with java option, -Dsbt.io.implicit.relative.glob.conversion=allow
```

**Comment by zsxwing** [comment]
Yep. These are new. But I don't know whether it could cause the issue of not. One thing I noticed is when the depth of a file is large, SBT will not copy it.

I submitted #266 to workaround the issue.

**Comment by zsxwing** [comment]
Fixed by #266
